docs/input.md

```diff
@@ -108,31 +120,119 @@ G15511    XX    1    D0ENMT    D0ENM_1    pathToFiles/D0ENMAC_1.bam
 G15511    XX    1    D0ENMT    D0ENM_2    pathToFiles/D0ENMAC_2.bam
 ```
 
-## Example TSV file for a normal/tumor pair with non recalibrated BAM files (step recalibrate)
+## Starting from the BAM prepare recalibration step
+
+To start from the preparation of the recalibration step (`--step prepare_recalibration`), a TSV file for a normal/tumor pair needs to be given as input containing the paths to the non recalibrated but already mapped BAM files.
+The TSV needs to contain the following columns:
+
+- `subject sex status sample bam bai`
+
+The same way, if you have non recalibrated BAMs and their indexes, you should use a structure like:
+
+```text
+G15511    XX    0    C09DFN    pathToFiles/G15511.C09DFN.md.bam    pathToFiles/G15511.C09DFN.md.bai
+G15511    XX    1    D0ENMT    pathToFiles/G15511.D0ENMT.md.bam    pathToFiles/G15511.D0ENMT.md.bai
+```
+
+When starting Sarek from the mapping step, a TSV file is generated automatically after the `MarkDuplicates` process.
+This TSV file is stored under `results/Preprocessing/TSV/duplicates_marked_no_table.tsv` and can be used to restart Sarek from the non-recalibrated BAM files.
+Using `--step prepare_recalibration` will automatically take this file as input.
+
+Additionally, individual TSV files for each sample (`duplicates_marked_no_table_[SAMPLE].tsv`) can be found in the same directory.
+
+If `--skip_markduplicates` has been specified, the TSV file for this step will be slightly different:
+
+```text
+G15511    XX    0    C09DFN    pathToFiles/G15511.C09DFN.bam    pathToFiles/G15511.C09DFN.bai
+G15511    XX    1    D0ENMT    pathToFiles/G15511.D0ENMT.bam    pathToFiles/G15511.D0ENMT.bai
+```
+
+When starting Sarek from the mapping step with `--skip_markduplicates`, a TSV file is generated automatically after the `Mapping` processes.
+This TSV file is stored under `results/Preprocessing/TSV/mapped.tsv` and can be used to restart Sarek from the non-recalibrated BAM files.
+Using `--step recalibrate --skip_markduplicates` will automatically take this file as input.
+
+Additionally, individual TSV files for each sample (`mapped_[SAMPLE].tsv`) can be found in the same directory.
+
+## Starting from the BAM recalibration step
+
+To start from the recalibration step (`--step recalibrate`), a TSV file for a normal/tumor pair needs to be given as input containing the paths to the non recalibrated but already mapped BAM files.
+The TSV needs to contain the following columns:
+
+- `subject sex status sample bam bai recaltable`
 
 The same way, if you have non recalibrated BAMs, their indexes and their recalibration tables, you should use a structure like:
 
 ```text
```

Comment:
That's a good question, I guess the easy answer is that we started copying the actual TSV files instead of making a nice table.
But I'll do that, it sounds like a good idea